salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -20,5 +20,23 @@ edx:
         roles:
           - edx
           - edx-{{ app_type }}
+
+  - edx-worker-{{ environment_name }}-{{ app_type }}-{{ id_num }}:
+      network_interfaces:
+        - DeviceIndex: 0
+          AssociatePublicIpAddress: False
```

Comment:
I'm under the impression that public IP is mainly for inbound traffic and outbound traffic for an instance without a public IP will go through the gateway. I was looking at some of my old terraform stuff and instances like rabbitmq weren't assigned a public IP since they only were accessed from the edxapp nodes. Maybe I'm missing something though and need to test it out in qa.